src/main/java/com/microsoft/aad/adal4j/WSTrustRequest.java

```diff
@@ -77,13 +77,52 @@ static WSTrustResponse execute(String url, String username,
         return WSTrustResponse.parse(response, policy.getVersion());
     }
 
+    static WSTrustResponse execute(String mexURL,
```

Comment:
It might be better to refactor this function since a lot of it is code duplication with existing execute(...) function